.travis.yml

```diff
@@ -4,5 +4,5 @@ dist: xenial
 python: "3.7"
 cache: pip
 before_install:
-- pip install -r requirements.txt
+- pip install pre-commit
```

Comment:
At some point, you're just gonna need to start using `tox` as a glue for having the same test env locally and in the CI...